tests/app/v2/notifications/test_post_notifications.py

```diff
@@ -1122,36 +1121,39 @@ def test_post_notifications_doesnt_save_email_to_queue_for_test_emails(client, n
     assert json_resp['id']
     assert json_resp['content']['body'] == "Dear citizen, have a nice day"
     assert json_resp['template']['id'] == str(template.id)
-    assert mock_send_task.called
-    assert not save_email_task.called
-    assert len(Notification.query.all()) == 1
+    # save email
+    save_email_task.assert_called_once_with([mock.ANY], queue='save-api-email-tasks')
+    mock_send_task.assert_called_once_with([json_resp['id']], queue='send-email-tasks')
+    assert Notification.query.count() == 1
 
 
-def test_post_notifications_doesnt_save_email_to_queue_for_sms(client, notify_db_session, mocker):
```

Comment:
Optional: If I understand what this original test is doing, it might be worth adding a test to make sure that a letter sent by a high volume service processes fine and doesn't use the save-api tasks